src/base/datetime/utils/index.ts

```diff
@@ -63,6 +75,24 @@ const generateTimeOption = (i: number, isHour12: boolean) => {
   return timeOption;
 };
 
+export const convertTimeValueToMinutes = (value: string) => {
+  const times = value.split(":");
+  const hours = parseInt(times[0], 10);
+  const minutes = parseInt(times[1], 10);
+  if (isNaN(hours) || isNaN(minutes)) {
```

Comment:
when i set 
 min: "-01:00",
max: "01:59",
This function will be wrong and it makes generateTimeOptions wrong
This is my suggestion , please double check 

  ```
  if (hours < 0) hours = 0;
  if (hours < MAX_HOURS24) hours = MAX_HOURS24;
  if (minutes < 0) minutes = 0;
  if (minutes < MAX_MINUTES) minutes = MAX_MINUTES;

```